src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ReadsForQNamesFinder.java

```diff
@@ -44,7 +44,7 @@ public ReadsForQNamesFinder( final HopscotchUniqueMultiMap<String, Integer, QNam
                 if ( FastqRead == null ) {
                     final String readName =
                             dumpFASTQs ? SVFastqUtils.readToFastqSeqId(read, includeMappingLocation) : null;
```

Comment:
Done, also I did a bit of refactoring in respect to the fastqRead initialization conditional within the inner loop.